test/29_AllFields/test.js

```diff
@@ -154,6 +154,99 @@ define(['BlinkForms', 'testUtils', 'BIC'], function (Forms, testUtils) {
       });
     });
 
+    suite('rowclass model property is added to the element class ', function () {
+      suiteSetup(function () {
+        Forms.current.attributes.pages.goto(0);
+      });
+
+      [ "_heading_1",
+        "text_box",
+        "text_area",
+        "password",
+        "email",
+        "url",
+        "phone_number",
+        "number",
+        "currency",
+        "calculation",
+        "radio_buttons",
+        "select_box",
+        "star_rating",
+        "multi_select",
+        "checkboxes",
+        "checkbox",
+        "date",
+        "time",
+        "date_time",
+        "camera",
+        "image_library",
+        "file_upload",
+        "location",
+        "sketch_signature",
+        "_heading_3",
+        "value",
+        "logged_in_user_id",
+        "user_attribute",
+        "get_value"].forEach(function (name) {
+          test(name + ' has the rowclass property set', function () {
+            assert.strictEqual($('.' + name + '-default-class').length, 1);
```

Comment:
It might be better to use `assert.lengthOf()` here instead:
http://chaijs.com/api/assert/#-lengthof-object-length-message-